packages/medusa/src/api/routes/admin/orders/request-return.js

```diff
@@ -141,6 +145,13 @@ export default async (req, res) => {
                 }
               }
 
+              let order = await orderService
+                .withTransaction(manager)
+                .retrieve(id)
+
+              const evaluatedNoNotification = value.no_notification !== undefined ? value.no_notification : order.no_notification
```

Comment:
Primary way of deciding on the value of the flag; If given - then use this value, otherwise inherit value from (mostly) order.